website/README.md

```diff
@@ -0,0 +1,39 @@
+# x11libre-website
+
+A website for the X11Libre (A fork of [Xorg XServer](https://gitlab.freedesktop.org/xorg/xserver)) project by @metux and the contributors
+
+## About
+
+X11Libre is an open-source initiative focused on providing a modern, libre implementation of the X11 protocol and related tools.
+
+## Features
+
+- Open-source and community-driven
+- Modern web technologies
+- Documentation and resources for developers
+
+## Getting Started
+
+To contribute or deploy the website:
+
+```bash
+git clone https://github.com/metux/x11libre-website.git
+cd x11libre-website
+# Follow project-specific setup instructions
+```
+
+## License
+
+This project is licensed under the MIT License.
+
+## Contact
+
+For questions or contributions, please open an issue or contact [@metux](https://github.com/metux).
+
+### Made by [@xgui4](https://github.com/xgui4) and [@ChocodeerZyNX](https://github.com/ChocodeerZyNX) and [@yusi1](https://github.com/yuzi1)
+
+```text
+before doing commit the dev of the site is done here : https://www.github.com/xgui4/X11Libre-website
+
+you can try the website at : https://xgui4.github.io/xserver/website
```

Comment:
These lines are valid only on your fork, but are not valid in the context of this PR.

You may use 2 different branches on your fork: one with these lines for the website deployed from your fork, and another one for this PR, with URLs that correspond to the X11Libre/xserver repo.